src/components/ReportActionItem/MoneyRequestPreview/MoneyRequestPreviewContent.tsx

```diff
@@ -130,27 +132,35 @@ function MoneyRequestPreviewContent({
     };
 
     const getPreviewHeaderText = (): string => {
+        let message = translate('iou.cash');
+
         if (isDistanceRequest) {
-            return translate('common.distance');
+            message = translate('common.distance');
+        }
+
+        if (isScanRequest) {
+            message = translate('tabSelector.scan');
+        }
+
+        if (isManualRequest) {
+            message = translate('tabSelector.manual');
```

Comment:
Isn't copy of manual request is "Cash" which is the production behavior?
If "Manual" then when is the case of "Cash"?